eos-chrome-plugin-update-upstream

```diff
@@ -21,6 +21,21 @@ DEST_DIR=/var/lib/chromium-plugins-extra/eos-chrome-plugin-updater
 WGET_COMMAND="wget -c --tries=5"
 DAYS_CHECK=7
 
+# Data needed for the Flash plugin, from Adobe
+FLASH_FALLBACK_VERSION="23.0.0.185"
+FLASH_FALLBACK_URL="https://fpdownload.adobe.com/pub/flashplayer/pdc/${FLASH_FALLBACK_VERSION}/flash_player_ppapi_linux.$(arch).tar.gz"
+FLASH_FALLBACK_SHA256SUM="3f7a13a615d3cf19e5feb02485e12e3785a6ef07cd7ae32072cd860468fd322b"
+
+FLASH_LKGV_FILE_URL="https://raw.githubusercontent.com/endlessm/eos-chrome-plugin-updater/master/flashplugin-$(arch).lkgv"
```

Comment:
I am not sure I fully understand the logic behind the `lkgv` file. Is it so that we can update the plugin version that is going to be downloaded without pushing out an OS update?
Pulling from github directly may be OK for now, but we probably want to host these files on S3 later on. We may also need to have different URLs depending on which branch/OS release we're on...
However, is there no APT index file on the Adobe servers we can piggyback on?